TopQuarkAnalysis/TopHitFit/src/Resolution.cc

```diff
@@ -171,6 +171,7 @@ namespace hitfit {
   //   The uncertainty for a momentum P.
   //
   {
+    p = std::fabs(p);
     if (_inverse)
       p = 1 / p;
```

Comment:
If it can be negative, then it could also be zero. Or not?

Reply:
@perrotta The kt components the MET resolution is parameterized in can be 0. I think however this inverse function is not used for the met, it is only used for the muons where the momentum input can't be zero.